services/src/main/java/io/druid/cli/PullDependencies.java

```diff
@@ -65,7 +65,38 @@
 
   private static final Set<String> exclusions = Sets.newHashSet(
       "io.druid",
-      "com.metamx.druid"
+      "com.metamx.druid",
+
+      // These should be included by druid and not pulled down in extensions
```

Comment:
More explicitly, this came about through exploration of myself and @hbutani on how to get https://github.com/metamx/druid-spark-batch/blob/master/build.sbt to work